### DISCUSSION TITLE
Any migration from extensions sync?

### DISCUSSION THREAD

**Discussion by NathanAlcantara** [vikdevelop/SaveDesktop#333] [Q&A]
I'm looking for any migrator from https://github.com/oae/gnome-shell-extensions-sync

I have all my configuration save on gist, I can't save anymore because I already installed a new gnome version (46) but this extension doesn't work anymore.

What you guys suggest? There's a way to convert my gists files to use this app?

**Comment by vikdevelop** [comment]
Hello, you can create the configuration archive by clicking on the Save button on the page with the same name, which would contain, except for the GSettings, your icons, themes, fonts, backgrounds, extensions, and even Flatpak apps and their user data. You can also set up synchronization with other computers using GNOME Online Accounts.
